Platforms/Docs/Common/debugging.md

```diff
@@ -1,5 +1,31 @@
 
-# Debugging QEMU
+# Debugger QEMU Platforms
+
+This document described different ways to debug on the QEMU platforms.
+
+## Debugging using UEFI debugger
+
+Both Q35 and SBSA are setup to debug debugged using the MU debugger package. for
+more details on using this debugger, see the [FeatureDebuggerPkg Readme](https://github.com/microsoft/mu_feature_debugger/blob/main/DebuggerFeaturePkg/Readme.md).
+
+By default the debugger is disabled, to enable you must both enable the debugger
+build flag to enable the source debug device state flag and specify a serial port
+TCP port.
+
+```
+stuart_build -c Platforms\QemuQ35Pkg\PlatformBuild.py BLD_*_DEBUGGER_ENABLED=TRUE SERIAL_PORT=5555 --flashrom
+```
+
+On Q35 this allows for debugging over a different port then the usual debug output
+because Q35 has a seperate serial port available to it. On SBSA the serial port
+will be shared with the logging output.

```

Comment:
To clarify, the debugger will use 5555 in your example, right? How would you access the Q35 serial output port? Or is that expected to just go to the console output?

Reply:
Yes, the normal debug output would go to the console like normal.